examples/knative/powershell/kn-ps-harbor-slack/README.md

```diff
@@ -0,0 +1,112 @@
+# kn-ps-harbor-slack
+Example Knative PowerShell function for sending to a Slack webhook when the Harbor webhook feature is configured to send repository events to the [kn-go-harbor-webhook function example](https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/development/examples/knative/go/kn-go-harbor-webhook).
+
+# Step 1 - Build
+
+
+Create the container image locally to test your function logic.
+
+```
+export TAG=<version>
+docker build -t <docker-username>/kn-ps-harbor-slack:${TAG} .
+```
+
+# Step 2 - Test
+
+Verify the container image works by executing it locally.
+
+Change into the `test` directory
+```console
+cd test
+```
+
+Update the following variable names within the `docker-test-env-variable` file
+
+* SLACK_WEBHOOK_URL - Slack webhook URL
+* SLACK_MESSAGE_PRETEXT - Text displayed for Slack notification
+
+Start the container image by running the following command:
+
+```console
+docker run -e FUNCTION_DEBUG=true -e PORT=8080 --env-file docker-test-env-variable -it --rm -p 8080:8080 <docker-username>/kn-ps-harbor-slack:${TAG}
+```
+
+In a separate terminal, run either `send-cloudevent-test.ps1` (PowerShell Script) or `send-cloudevent-test.sh` (Bash Script) to simulate a CloudEvent payload being sent to the local container image
+
+```console
+Testing Function ...
+See docker container console for output
+
+# Output from docker container console
+Detected change to subject-123 ...
```

Comment:
question: is this really an output from your function? "subject-123" doesn't seem like?

Reply:
It's not an output from my function. I simply reused the content from our [Slack function example](https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/development/examples/knative/powershell/kn-ps-slack).

Reply:
so it needs to be updated, right?

Reply:
Updated